src/Screen/Assignment1/Assignment1.tsx

```diff
@@ -0,0 +1,48 @@
+import { useState } from "react";
+import Footer from "../../Components/Footer";
+import {
+  CounterButton,
+  Div,
+  CounterButtonDiv,
+} from "./CounterStyledComponents/Assignment1StyleComponent";
+
+const Assignment1 = () => {
+  const [count, setCount] = useState(0);
+
+  const increaseCount = () => {
+    setCount(count + 1);
+  };
+  const decreaseCount = () => {
+    if (count > 0) {
+      setCount(count - 1);
+    }
+  };
+  const resetCount = () => {
+    setCount(0);
+  };
+  return (
+    <>
+      <Div>
```

Comment:
Please use different names of style components i.e. not related to existing html tags